packages/nocodb/src/lib/cache/NocoCache.ts

```diff
@@ -77,10 +82,15 @@ export default class NocoCache {
   public static async appendToList(
     scope: string,
     subListKeys: string[],
+
     key: string
   ): Promise<boolean> {
     if (this.cacheDisabled) return Promise.resolve(true);
-    return this.client.appendToList(scope, subListKeys, key);
+    return this.client.appendToList(
+      scope,
+      subListKeys,
```

Comment:
Does it require prefixes for each sublist key?

Reply:
It's been covered inside `appendToList` before `set`.